servers/rendering/shader_language.cpp

```diff
@@ -6110,7 +6110,7 @@ ShaderLanguage::Node *ShaderLanguage::_parse_expression(BlockNode *p_block, cons
 					String at;
 					for (int j = 0; j < op->arguments.size(); j++) {
 						if (j > 0) {
-							at += " and ";
+							at += RTR(" and ");
```

Comment:
This is tricky for translation, there are good chances that translators might remove the spaces by mistake, and it's not sure that `<arg0> and <arg1> and <arg2>` is something that can be translated in the same order for any language (for example some languages might have constructs like `<arg0> <arg1>-and <arg2>-and`? Not that I know any like this but typically anything we can think of might exist somewhere :)).

So this might be worth refactoring a bit. Ideally there shouldn't be any English text in that `at` I think. Maybe just:
```suggestion
							at += ", ";
```